### PR TITLE
feature/explore-assets

### PR DESCRIPTION
**Pull request by DmitriVanGuard**
### Summary
- New mobile assets page;
- Recently viewed assets and watchlists (only last 5);

### Screenshots
image

### CODE REVIEW

File: src/pages/assets/AssetsOverview.module.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 @import '~@santiment-network/ui/variables.scss';
-@import "~@santiment-network/ui/mixins.scss";
-@import "../../mixins.scss";
+@import '~@santiment-network/ui/mixins.scss';
+@import '../../mixins.scss';
```
Comment (haritonasty) [inline src/pages/assets/AssetsOverview.module.scss]: only mixins from sun-ui maybe is enough

File: src/components/Watchlists/Watchlist.module.scss [hunk 2]
```diff
@@ -53,3 +57,9 @@
   align-items: center;
   width: 200px;
 }
+
+.subtitle {
+  @include text('body-1', 'm');
+
+  margin: 32px 16px 12px;
+}
```
Comment (haritonasty) [inline src/components/Watchlists/Watchlist.module.scss]: too big space between `Asset Overview` and `Categories` when recently is empty
image
maybe set `margin-bottom` for recently block

Comment (DmitriVanGuard) [reply src/components/Watchlists/Watchlist.module.scss]: image

**Comment by DmitriVanGuard** [comment]
@haritonasty Updated long name handling and other stuff: 
image
image

**Pull request merged**